config/packages/serializer/Centreon/Acknowledgement.Acknowledgement.yml

```diff
@@ -0,0 +1,50 @@
+Centreon\Domain\Acknowledgement\Acknowledgement:
```

Comment:
Could you move all serializer files to "yaml" extension (instead of yml) to let it consistent with other

Reply:
it's a requirement by the bundle

> If you are using YAML files as metadata format, the file extension to use is .yml

https://jmsyst.com/bundles/JMSSerializerBundle/master/configuration#defining-metadata